From 0d28842ecedad353cc5af9acf583be0f5ec71dfe Mon Sep 17 00:00:00 2001
From: Alexander Bokovoy <abokovoy@redhat.com>
Date: Mon, 15 Oct 2012 16:01:26 +0300
Subject: [PATCH] Remove bogus check for smbpasswd

We don't use smbpasswd when configuring IPA for AD trusts anymore
because we switched to use Kerberos authentication in IPA passdb
backend based on CIFS service keytab.

https://fedorahosted.org/freeipa/ticket/3181
---
 ipaserver/install/adtrustinstance.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ipaserver/install/adtrustinstance.py b/ipaserver/install/adtrustinstance.py
index b74f4b685b1cb56bdcdbae15ab6a8f38960bfc66..b3602ddce7fa1c184bdf1283e67b50463e49c5fd 100644
--- a/ipaserver/install/adtrustinstance.py
+++ b/ipaserver/install/adtrustinstance.py
@@ -60,9 +60,9 @@ and re-run ipa-adtrust-instal again afterwards.
 """
 
 def check_inst():
-    for smbfile in ['/usr/sbin/smbd', '/usr/bin/net', '/usr/bin/smbpasswd']:
+    for smbfile in ['/usr/sbin/smbd', '/usr/bin/net']:
         if not os.path.exists(smbfile):
-            print "%s was not found on this system" % file
+            print "%s was not found on this system" % smbfile
             print "Please install the 'samba' packages and " \
                   "start the installation again"
             return False
-- 
1.6.0.6

